e2e/actions/actions.go

```diff
@@ -250,6 +250,12 @@ func (c *actionTests) actionShell(t *testing.T) {
 			name: "ShellExit",
 			argv: []string{c.env.ImagePath},
 			consoleOps: []e2e.SingularityConsoleOp{
+				// "cd /" to work around issue where a long
+				// working directory name causes the test
+				// to fail because the "Singularity" that
+				// we are looking for is chopped from the
+				// front.
```

Comment:
Could you ad as TODO the commit hash that introduced the error, just to track it, and be able to circle back in the future and fix it.